app/src/main/AndroidManifest.xml

```diff
@@ -64,6 +64,10 @@
             </intent-filter>
         </service>
 
+        <meta-data
+            android:name="com.google.android.geo.API_KEY"
+            android:value="AIzaSyAV5713sUQ-j8KxDjuPGtyVq1aQY1iJkuY" />
```

Comment:
We can't expose this api key to the public. In fact is this key is already referenced in build config and could the line be replaced with:  `"${API_KEY}"`

Can you also remove the 3 other occurrences of this key across the YouTube* classes